Jellyfin.Server/Migrations/MigrationOptions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Jellyfin.Server.Migrations
+{
+    /// <summary>
+    /// Configuration part that holds all migrations that were applied.
+    /// </summary>
+    public class MigrationOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MigrationOptions"/> class.
+        /// </summary>
+        public MigrationOptions()
+        {
+            Applied = System.Array.Empty<string>();
+        }
+
+#pragma warning disable CA1819 // Properties should not return arrays
+        /// <summary>
+        /// Gets or sets he list of applied migration routine names.
```

Comment:
```suggestion
        /// Gets or sets the list of applied migration routine names.
```

Reply:
Why didn't we use an integer to track the current migration again?

Reply:
Coordination between branches if multiple migrations need to be worked on at the same time

Reply:
Ah right, very good reason.